zeroize/derive/src/lib.rs

```diff
@@ -253,277 +280,278 @@ impl ZeroizeAttrs {
     }
 }
 
-fn generate_fields(s: &mut synstructure::Structure<'_>, method: TokenStream) -> TokenStream {
-    s.bind_with(|_| BindStyle::RefMut);
+fn field_ident(n: usize, field: &Field) -> Ident {
+    if let Some(ref name) = field.ident {
+        name.clone()
+    } else {
+        format_ident!("__zeroize_field_{n}")
```

Comment:
That seems odd... perhaps this?

```suggestion
        format_ident!("__zeroize_field_{}", n)
```